packages/svelte-vscode/package.json

```diff
@@ -113,9 +113,15 @@
                 "svelte.plugin.typescript.selectionRange.enable": {
                     "type": "boolean",
                     "default": true,
-                    "title": "TypeScript: SelectionRange",
+                    "title": "TypeScript: Selection Range",
                     "description": "Enable selection range for TypeScript"
                 },
+                "svelte.plugin.typescript.signatureHelp.enable": {
```

Comment:
Wondering if we should use the LSP name in the config or just `parameter hints`, maybe that's more understandable.

Reply:
I prefer the LSP name just because they put some thought into the name (I hope 😄 ) so I would go with it.